recipes/r-essentials/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set name = "r-essentials" %}
+{% set version = "1.5.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+    number: 0
+    string: r3.3.2_0
```

Comment:
In theory not. But I took this from the r-channel and they did it that way. Any recommendations welcome.